xml/System.IO/StreamReader.xml

```diff
@@ -1008,7 +1008,7 @@
         <exception cref="T:System.ArgumentNullException">
           <paramref name="stream" /> or <paramref name="encoding" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentOutOfRangeException">
-          <paramref name="bufferSize" /> is less than or equal to zero.</exception>
+          <paramref name="bufferSize" /> is less than or equal to zero, except for -1, which is allowed in stream-based constructors to indicate the default buffer size.</exception>
```

Comment:
After https://github.com/dotnet/runtime/pull/118958, this no longer applies to stream-based ctors.
```suggestion
          <paramref name="bufferSize" /> is less than or equal to zero, except for -1, which is allowed to indicate the default buffer size.</exception>
```